code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -592,7 +592,8 @@ GLOBAL_LIST_INIT(runed_metal_recipes, list ( \
 		return
 	var/turf/T = get_turf(user) //we may have moved. adjust as needed...
 	var/area/A = get_area(user)
-	if((!is_station_level(T.z) && !is_mining_level(T.z)) || (A && !(A.flags_1 & CULT_PERMITTED_1)))
+	if((!is_station_level(T.z) && !is_mining_level(T.z)) || (A && !(A.flags_1 & CULT_PERMITTED_1) || !(A.area_flags & VALID_TERRITORY | BLOBS_ALLOWED) || ! istype(A, 
+/area/service/chapel)))
```

Comment:
I believe the logic linking of this is slightly wrong, shouldn't it be more something like
```suggestion
	if((!is_station_level(T.z) && !is_mining_level(T.z)) || (A && ((!(A.flags_1 & CULT_PERMITTED_1) && !(A.area_flags & VALID_TERRITORY | BLOBS_ALLOWED)) || istype(A, 
/area/service/chapel))))
```

Reply:
@silicons mind looking into that? ^

Reply:
`A.area_flags & VALID_TERRITORY | BLOBS_ALLOWED` will never be `0` because of the order of operations. gotta do `A.area_flags & (VALID_TERRITORY | BLOBS_ALLOWED)` instead